packages/circuit-ui/components/Selector/Selector.stories.tsx

```diff
@@ -51,4 +53,21 @@ export const Disabled = (args: SelectorProps) => (
   <Selector {...args}>I cannot be selected</Selector>
 );
 
+export const Sizes = (args: SelectorProps) => (
+  <Stack>
+    <Selector {...args} size="kilo">
+      Kilo
+    </Selector>
+    <Selector {...args} size="mega">
+      Mega
+    </Selector>
+    <Selector {...args} size="flexible">
+      <Text bold noMargin>
```

Comment:
This will need to be `<Body />` in V3. Not sure how we should handle this since the typography PR (https://github.com/sumup-oss/circuit-ui/pull/884) was already merged 🤔 Thoughts @connor-baer?

Reply:
It should be fine if I make this whole change non-breaking, right?

Reply:
Yes it's fine 🙁 We'll go over the codebase again before v3 and address any leftover typography changes